### DISCUSSION TITLE
Is the conda package creation workflow working successfully?

### DISCUSSION THREAD

**Discussion by sugatoray** [deepchecks/deepchecks#559] [Q&A] [answered]
I would like to know if the conda package creation workflow works or not. Thank you.

cc: @ItayGabbay

**Comment by ItayGabbay** [comment]
Hello @sugatoray 
We currently have a conda creation workflow but we trigger it manually at the moment. you can view it under the .github/workflows folder.
We do plan to trigger a run on an automatic release starting with the 0.4 milestone.

**Reply by sugatoray** [reply]
Thank you. I had looked at the workflows. And tried using (on a repository of mine) the conda-publish workflow. But it failed to upload to a channel. It requires automatic upload to be set in `.condarc` file. And I could not find a way to specify that in the GitHub action you are using in the workflow.

**Comment by ItayGabbay** [comment, marked as answer]
In the end the packaging to conda-forge uses this repository: https://github.com/conda-forge/deepchecks-feedstock
